### PR TITLE
added tag methods to contact

### PR DESCRIPTION
**Pull request by toreym**
added tag methods.
also configure tlsv1 option explicitly for servers that have disabled sslv3.

### CODE REVIEW

**Comment by mhenrixon** [comment]
Hmm I wonder why I didn't get this email

File: lib/active_campaign/client.rb [hunk 1]
```diff
@@ -61,6 +61,7 @@ def create_request(method, api_method, options = {})
 
         request.headers              = { "User-Agent" => user_agent }
         request.auth.ssl.verify_mode = :none
+        request.auth.ssl.ssl_version = :TLSv1
```
Comment (mhenrixon) [inline lib/active_campaign/client.rb]: Will this work for everyone? For us it didn't or it was sslv3 that didn't work. Never the less I have a feeling we should make this configurable and also add some information about it in the readme?

**Comment by vmotsak** [comment]
@mhenrixon can you merge tags methods, while you decide on ssl_version?

**Comment by mhenrixon** [comment]
Sorry @vmotsak I'll see what can be done!

**Pull request merged**